src/networking.c

```diff
@@ -319,6 +319,46 @@ int prepareClientToWrite(client *c) {
     return C_OK;
 }
 
+/* Returns everything in the client reply linked list in a SDS format.
+ * This should only be used only with a caching client. */
+static sds getClientOutputBuffer(client *c) {
```

Comment:
Technically speaking, we are not getting the original buffers so maybe consider `aggregateClientOutputBuffer`?

```suggestion
static sds aggregateClientOutputBuffer(client *c) {
```